selftests/functional/test_output.py

```diff
@@ -459,6 +459,15 @@ def test_broken_pipe(self):
         self.assertIn("not found", result.stderr)
         self.assertNotIn("Avocado crashed", result.stderr)
 
+    def test_results_plugins_no_tests(self):
+        os.chdir(basedir)
+        for output in ['json', 'xunit', 'tap']:
+            cmd_line = ("%s run UNEXISTING --job-results-dir %s --%s -"
+                        % (AVOCADO, self.tmpdir, output))
+            result = process.run(cmd_line, ignore_status=True)
```

Comment:
We do generate all of the results by default, how about running the command just once and then checking all results?